app.js

```diff
@@ -0,0 +1,77 @@
+
+import {
+    todoInputBox$,
+    todoList$,
+    todoSubmitButton$
+} from "./domElements.js";
+
+
+let todos = [];
+
+const createInputElement = (todo) => {
+
+    const inputElement = document.createElement('input');
+    inputElement.classList.add("text");
+    inputElement.type = "text";
+    inputElement.value = todo.text;
+    inputElement.setAttribute("readonly", "readonly")
+
+    return inputElement;
+}
+
+const handleDeleteTodo = (todo) => {
+    todoList$.innerHTML = null;
+    console.log(todo);
+    todos = todos.filter(todoElement => todoElement.id !== todo.id);
+    renderTodos();
+}
+
+const createTodoDeleteButton = (todo) => {
```

Comment:
on function naming we don't need to add `todo` everywhere as we already know what the project is. Rather we can say `onCreateDeleteButton`

Reply:
okay bhai. got it